package.json

```diff
@@ -27,6 +27,23 @@
   },
   "homepage": "https://github.com/IOMechs/RNColorPalette#readme",
   "devDependencies": {
-    "metro-react-native-babel-preset": "^0.58.0"
+    "@babel/core": "^7.1.2",
```

Comment:
@aliasgharkarani you sure all of these packages are required for testing? I.e. we don't have to remove any of these.

Reply:
extra packages removed.